build/myget-release.yml

```diff
@@ -153,10 +160,4 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
-          - task: NuGetCommand@2
-            displayName: 'Push to MyGet.org'
-            inputs:
-              command: push
-              packagesToPush: 'src/**/*.nupkg'
```

Comment:
Would this not be a argument that we should pass in?

Reply:
It's defaulting to that, but you can override if there is a need.

Can add it for sake of being explicit if you want

Reply:
Ah, ok, nevermind then.